Dockerfile

```diff
@@ -47,6 +47,7 @@ RUN rm -rf third_party/differential-privacy/java && \
 RUN \
     pipenv run bazel build src/python:bindings_test  --verbose_failures && \
     cp -f ./bazel-bin/src/bindings/pydp.so ./pydp && \
+    rm -rf dist/ && \
```

Comment:
Can we add a comment on why are we doing this?

Reply:
delete the dist folder inside the docker so that fresh .whl are built when we try to run the docker, this sounds okay ?

Reply:
yeah